### PR TITLE
Fix support for multiple generated sprites in local

### PR DESCRIPTION
**Pull request by Vorlias**
If tarmac generates multiple spritesheets, they would overwrite previous spritesheets with `--target local` due to having the same sprite names (`spritesheet`). 

This addresses that and suffixes each spritesheet with a number that's incremented per spritesheet.

E.g. two spritesheets -> `spritesheet-1`, `spritesheet-2`.

### CODE REVIEW

**Comment by jeparlefrancais** [comment]
Hey! Sorry I need to check my notification settings, I didn't even know you submitted a PR 😬 Thanks for your contribution it's awesome!

That seems like a reasonable fix! Can you run rustfmt so that CI checks are green? You can run `cargo fmt` and it should fix it.

Again, sorry the delay! Next time ping me on discord if I don't reply a few days

**Comment by Vorlias** [comment]
Looks like it's good now.

File: src/commands/sync.rs [hunk 1]
```diff
@@ -369,10 +374,8 @@ impl SyncSession {
 
         log::trace!("Alpha-bleeding {} packed images...", packed_images.len());
 
-        for (i, packed_image) in packed_images.iter_mut().enumerate() {
+        for (i, _packed_image) in packed_images.iter_mut().enumerate() {
             log::trace!("Bleeding image {}", i);
-
-            alpha_bleed(&mut packed_image.img);
```
Comment (jeparlefrancais) [inline src/commands/sync.rs]: Hum could this be a bad merge issue? This was added in the last PR

**Review by jeparlefrancais — CHANGES_REQUESTED** [review]

**Comment by Vorlias** [comment]
I'm not sure how that vanished, should be fixed now. Apologies for the _very late_ response. Been quite busy. + yeah, github is terrible at telling you this stuff.

**Pull request closed without merging**